### DISCUSSION TITLE
Show damage tooltip even if the target is not accessible in one move

### DISCUSSION THREAD

**Discussion by VanHexing** [ihhub/fheroes2#7575] [Ideas]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

It is convenient to see how many enemy creatures my unit will kill once it reaches them (even if it cannot reach them in this move).

### Describe the possible solution

When pointing the mouse to an enemy creature that cannot be reached by my active unit in one turn, it might still produce the same damage tooltip as if the enemy creature can be reached (when the mouse pointer turns into a sword).

**Comment by ihhub** [comment]
Hi @VanHexing , this might confuse some players. For instance, I am new player of HoMM2 and I toggled this option. I see the popup window with damage info. That's cool. My first thought is that I can hit that monster. I click at the unit and nothing happens. A bug I might think? Certainly yes.

**Comment by ihhub** [comment]
Moving this issue into the discussion section.
